FF1Lib/ItemPlacement.cs

```diff
@@ -513,6 +513,8 @@ protected override ItemPlacementResult DoSanePlacement(MT19337 rng, ItemPlacemen
 			{
 				placementFailed = false;
 
+				var balancedPicker = new RewardSourcePicker(0.5, 8.5, _checker);

```

Comment:
Should there be another flag to control whether NPCs have boosted weight vs chests?  and where did the weight of 8.5 come from?

Reply:
i have integrated it into the spreadPlacement Flag.

Reply:
That number is a "tuned" parameter. I divided the number of chests by the number of npcs. Took half of that and looked through some spoiler logs to see if it was too high or too low.
I looked through some spoiler logs again and lowered it a little.

Reply:
ok, I suggest taking this reply you just wrote here and putting it in a comment in the code.

Reply:
ok